plugins/cassandra/cassandrawu.cpp

```diff
@@ -3554,6 +3554,13 @@ class CCasssandraWorkUnitFactory : public CWorkUnitFactory, implements ICassandr
                     merger->addResult(*new CassandraResult(fetchDataForWuid(workunitInfoMappings, value, true)));
                 }
             }
+            else if (!thorTimeThreshold.isEmpty())
```

Comment:
this doesn't look right to me, because in lines 3569/3570, it is also using thorTimeThreshold,
but now, with this change, if set, it will never reach there, even if sortByThorTime is true.

Reply:
@jakesmith the lines 3569/3570 will be reached if sortByThorTime is true, but, the thorTimeThreshold is empty.

Reply:
@wangkx , I was talking about the case where thorTimeThreshold is set and sortByThorTime is true.

With this change it will not follow the continuation code inside the 'if (wuidFilters.length())' block

Reply:
Make sense. I may remove my changes (line 3557-3563) and change line 3564 to:
else if (sortByThorTime || !thorTimeThreshold.isEmpty())
The wuidFilters should be used when !thorTimeThreshold.isEmpty() even if sortByThorTime  is false. Any objection?